readme.js

```diff
@@ -1,27 +1,35 @@
 class ReadmeComponent extends HTMLElement {
   constructor() {
     super();
-    this.attachShadow({ mode: "open" });
+    this.attachShadow({mode: "open"});
     this.path = this.getAttribute("path") || "readme";
   }
 
   connectedCallback() {
-    this.path = this.getAttribute("path") || "readme";
-    this.loadScript(
-      "https://cdnjs.cloudflare.com/ajax/libs/showdown/1.9.1/showdown.min.js"
-    ).then(() => {
-      this.fetchReadme();
-    });
+    Promise.all([
```

Comment:
My ignorance not a suggestion: Is there a difference between `Promise.all([onePromise]).then` and `onePromise.then`?

Reply:
Eh, not right now but when I make the mermaid thing work there's gonna be another promise and I'm gonna want an array of those resolved values, so it's just my mental placeholder